integration-tests/src/test/java/oracle/weblogic/kubernetes/TestConstants.java

```diff
@@ -143,11 +143,11 @@
   public static final String K8S_NODEPORT_HOST = Optional.ofNullable(System.getenv("K8S_NODEPORT_HOST"))
         .orElse(assertDoesNotThrow(() -> InetAddress.getLocalHost().getHostAddress()));
   public static final String RESULTS_ROOT = System.getenv().getOrDefault("RESULT_ROOT",
-      System.getProperty("java.io.tmpdir")) + "/ittestsresults";
+      System.getProperty("java.io.tmpdir")) + "/it-testsresults";
```

Comment:
IMO, we should place all IT logs under one directory. So that it is easy to delete one directory when cleaning up instead of trying to delete multiple.
We should create /<tmp>/it-tests/  as base dir and then use this for all other logs.

Reply:
Do u want to merge it-diagnosticlogs and it-ittestsresults into one directory under it-logs  ?

Reply:
Yes.

Reply:
How about following directory structure proposal ... 

public static final String RESULTS_ROOT = System.getenv().getOrDefault("RESULTS_ROOT",
      System.getProperty("java.io.tmpdir")) + "/it-testresults";

public static final String LOGS_DIR =  RESULTS_ROOT  + "/diagnotsics";
public static final String PV_ROOT =   RESULTS_ROOT + "/pvroot";
public static final String WORK_DIR = RESULTS_ROOT + "/workdir";

So that we have only one Environment Variable  RESULTS_ROOT to deal with.

Reply:
commit the change to the PR

Reply:
That seems good.